api/index.js

```diff
@@ -17,86 +18,71 @@ var utils = require('../utils');
 var app = express();
 
 const iberaServicesEndpoint = config.IBERA_SERVICES_ENDPOINT;
-const azureStorageConnectionString = config.AZURE_OI_STORAGE_CONNECTION_STRING;
+const azureStorageConnectionString = config.STORAGE_CONNECTION_STRING;
 
+const USER_TOKEN_HEADER_KEY = 'user-token';
 const CONTAINER_NAME = 'attachments';
 
+var azureBlobService = azureStorage.createBlobService(azureStorageConnectionString);
+
+//Will be implemented next,  https://amitu-ted.visualstudio.com/DefaultCollection/iBera/_workitems/edit/11
 async function getUserId(userToken) {
   return 'demo-user-001';
-  /* try {
-     var uri = documentServicesEndpoint + `/api/user`;
-     var result = await request({
-       method: 'GET',
-       uri,
-       headers: { 'User-Token': userToken },
-       json: true
-     });
-     console.log(`got response: ${util.inspect(result)}`);
-     return result;
-   }
-   catch (err) {
-     var errorMessage = `Could not retrieve userId from user token: ${userToken}`;
-     console.log(errorMessage);
-     throw new Error(errorMessage);
-   }
-   */
 }
 
 
-app.get('/config', async(req, res) => {
-
+app.get('/config', async (req, res) => {
   try {
-    var result = {
-      documentServiceUrl: config.DOCUMENT_SERVICES_ENDPOINT
-    }
+    var result = {documentServiceUrl: config.DOCUMENT_SERVICES_ENDPOINT}
 
     console.log(`sending configuration: ${util.inspect(result)}`);
     res.json(result);
   } catch (err) {
-    return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({
-      error: err.message
-    });
+    return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({error: err.message});
   }
 });
 
 app.post('/attachment', async(req, res) => {
-  if (!req.headers['user-token']) {
-    return res.status(HttpStatus.BAD_REQUEST).json({
-      error: `user-token request header is missing`
-    });
+  var userToken = req.headers[USER_TOKEN_HEADER_KEY];
+  if (!userToken) {
+    return res.status(HttpStatus.BAD_REQUEST).json({error: CONTAINER_NAME + ` request header is missing`});
+  }
+
+  if(!validate(req.body, schema.attachment.put).valid){
```

Comment:
this should be under schema.attachment.post (not put).
the post / put represent the HTTP method that the client invokes on the attachment resource

Reply:
Yes I changed it..